content/release.rst

```diff
@@ -197,3 +210,78 @@ Sample release notes
     * Fix an internal server error when FTP client requests
       an unknown command. [#160][ftp][ftps]
 
+
+Version Management
+==================
+
+Chevah release versions are based on MAJOR.MINOR.PATCH
+`Semantic Versioning specification <http://semver.org/>`_.
+
+A MAJOR version is released to introduce new features,
+remove functionalities which have become obsolete, or add features not
+compatible with previous versions.
+
+MINOR versions are released based on a rolling update development model at
+intervals varying between 30 to 60 days.
+The goal is to have functionalities and defect fixes available to customers as
+soon as possible.
+Each release has a certain overhead, and the overhead should be minimize by
+automating the release process.
+In an ideal works a release should be done by preparing a release branch
+and then by issuing a single command the documentation, download and news pages
+are updated, while users are automatically notified about the new releases.
```

Comment:
Always avoid such long phrases, they are uncommon in English. So:

_In an ideal **world** a release should be done by preparing a release branch. Then, by issuing a single command, the documentation, download and news pages will be updated. Users will be automatically notified about the new releases._